test/promql/tests/dep-resource-min-1m.yaml

```diff
@@ -0,0 +1,61 @@
+# This is the main input for unit testing.
+# Only this file is passed as command line argument.
+
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  # Test 1.
```

Comment:
Instead of a generic comment like this, please use a description explaining what this test does in human language. If you prefer to put the descriptions above each `expr` block, that's fine too.